### ISSUE TITLE
How do I make a footnote rule RTL in eledmac?

### ISSUE THREAD

**Issue by talazem** [maieul/ledmac#343]
In eledmac, how do I get the the footnote rule to be RTL, i.e. to begin at the right margin, as opposed to the left? I am using polyglossia and the bidi packages with Memoir, and eledmac for its critical edition apparatus. Below is an example demonstrating the problem. Notice:

 - the regular LaTeX/Memoir footnote correctly implements the invoked `\rightfootnoterule`
 - the two footnote commands specific to eledmac -- `\footnoteA` and `\edtext{...}{\Afootnote{...}}` do not implement the `\rightfootnoterule` command.

I'd be grateful for any help.

    \TeXXeTstate=1
    
    \documentclass[11pt,a4paper]{memoir}
    
    \usepackage{fontspec}
    \usepackage{polyglossia}
    \usepackage[series={A}]{eledmac}
    \setmainlanguage{arabic}
    \usepackage{bidi}
    
    \newfontfamily{\arabicfont}[Script=Arabic,Scale=1.2,WordSpace=1.66]{Arial Unicode MS}    
    
    \rightfootnoterule
    
    %%% DOCUMENT %%%
    
    \begin{document}
    
    \beginnumbering
    \pstart
    
     بسم الله الرحم الرحيم وصلى الله على سيدنا محمد وآله وصحبه وسلم الحمد لله المتفرد بالملك والملكوت . المتوحد بالعظمة والجبروة . 
    \footnote{هذه تعليقة.}
      يحي ويميت وهو حي لا يموت . وصلى الله وسلم على سيدنا محمد سيد السابقين والآخرين ورضي الله عن الصحابة والتابعين . وتابعيهم بإحسان إلى يوم الدين . وبعد فهذه الورقات سميتها .  بسم الله الرحم الرحيم وصلى الله على سيدنا محمد وآله وصحبه وسلم الحمد لله المتفرد بالملك والملكوت . المتوحد بالعظمة والجبروة .
    \footnoteA{هذه تعليقة.}
      يحي ويميت وهو حي لا يموت . وصلى الله وسلم على سيدنا محمد سيد السابقين والآخرين ورضي الله عن الصحابة والتابعين . وتابعيهم بإحسان إلى يوم الدين . وبعد فهذه الورقات سميتها . 
    بسم الله الرحم الرحيم وصلى الله على سيدنا محمد وآله وصحبه وسلم الحمد لله المتفرد بالملك والملكوت . المتوحد بالعظمة والجبروة .
    \edtext{هذه تعليقة}{\Afootnote{هذه تعليقة طويلة}}
      يحي ويميت وهو حي لا يموت .
      
    \pend
    \endnumbering
    
    \end{document}

Over on tex.stackexchange, Salim Bou states that there is no support from bidi to eledmac, and that I should redefine footnoterules manually. It seems that eledmac already recognised when to switch the bracket of the lemma in light of RTL, so perhaps the ability to implement bidi's footnote rules (or to achieve the same results) might be added? 

In the meantime, if there are any suggestions as to how I can achieve this by manually redefining footnoterules, I'd be grateful.

**Comment by maieul** [comment]
> 
> 
> In the meantime, if there are any suggestions as to how I can achieve this by manually redefining footnoterules, I'd be grateful.


If I would have such suggestion, I would be directly implemented in (r)eledmac ;-)

**Comment by talazem** [comment]
Fair enough. :-) I look forward to such a solution appearing. Until then, it seems I might not be able to use eledmac at this moment for the present critical edition. I will keep an eye on developments here, and will happily test out any proposed solutions. Thank you.

**Comment by maieul** [comment]
maybe @ralessi has an idea?

**Comment by talazem** [comment]
Salim Bou has posted a temporary solution over at http://tex.stackexchange.com/questions/255087/how-do-i-make-a-footnote-rule-rtl-in-eledmac/255149#255149. Perhaps it will help?

**Comment by maieul** [comment]
thanks for the link. for now, use it. I will look for a better way to implement it in futur...

**Comment by talazem** [comment]
Great, many thanks. As to this temporary solution's limitations, Salim Bou commented at that link: "For example the above code does not give the result in case of two columns footnote."
